develop/potions.md

```diff
@@ -0,0 +1,77 @@
+---
+title: Potions
+description: Adding a custom potion.
+authors:
+  - dicedpixels
+  - PandoricaVi
+  - Drakonkinst
+---
+
+# Potions
+
+Potions are consumables that grants an entity an effect. A player can brew potions using a Brewing Stand or obtain them
+as items through various other game mechanics.
+
+## Custom Potions
+
+Adding a potion follows a similar path as adding an item. You will create an instance of your potion and register it by
+calling `BrewingRecipeRegistry.registerPotionRecipe`.
+
+::: info
+When Fabric API is present, `BrewingRecipeRegistry.registerPotionRecipe` is made accessible through an Access Widener.
+
+You can directly call `BrewingRecipeRegistry.registerPotionRecipe` without using a mixin.
+:::
+
+Let's assume you don't have Fabric API and instead rely on a mixin invoker to access
```

Comment:
I think this should be avoided - it'll cause some confusion and people definitely skim read the callout and miss that this bit is completely optional.

Instead you should assume developers have Fabric API and put a callout briefly explaining how to do it without, eg: "If you do not have Fabric API, you can create a mixin invoker, or use access wideners."

Reply:
I moved the mixin accessor route to a separate section. Main example now uses the FAPI approach.